components/inputs/input-text.js

```diff
@@ -197,6 +197,10 @@ class InputText extends InputInlineHelpMixin(PropertyRequiredMixin(FocusMixin(La
 				:host([value-align="end"]) {
 					--d2l-input-text-align: end;
 				}
+				#d2l-input-text-container {
+					display: inline;
```

Comment:
I'm assuming the `display: inline` became necessary as a result of changing the position. Ya? Though I'm not sure why.